app/assets/javascripts/image_question_drawing_tool.js.coffee

```diff
@@ -74,6 +73,10 @@ class ImageQuestionDrawingTool
     @$annotation_field          = $("#{@form_sel} [name=\"#{@form_prefix}[annotation]\"]")
     @$answer_text_field         = $("#{@form_sel} [name=\"#{@form_prefix}[answer_text]\"]")
 
+    # Find the first interactive which is on the same page (aka content-module, div.content-mod) as the question.
+    # It's necessary when we are in single page rendering mode and there are multiple interactives rendered.
+    @interactive_element = @$main_form.closest('.content-mod').find('.interactive-mod > *:first-child')[0]
```

Comment:
Instead of finding the interactive like this. Could the ruby code determine the id of interactive on the page and pass that that into here?
We hopped at some point to support multiple interactives on a page even in 'multiple page' layout. In that case the author would need to explicitly select which interactive an image question is connected to. To move in that direction, the ImageQuestion rails model could have a method which returns the the first interactive that is on the same page. And that would be eventually be passed passed into this code in some form.

Then when we really add support for multiple interactives on a single page (in multiple page layout), we would just need to change the ImageQuestion rails model and this code could remain the same.